CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.26.1]
```

Comment:
I think this probably should be `4.25.10`

Reply:
Updated as per suggestion, thank you